src/client/components/logout/logout.style.css

```diff
@@ -100,9 +92,12 @@
   padding: 0.3em 3em;
   text-align: center;
   text-transform: uppercase;
-
 }
 
 .confirm-cancel:active {
   box-shadow: var(--debossed-shadow-medium);
 }
+
+.sb-show-main .popup-content {
+  position: absolute !important;
```

Comment:
Do we really need `!important` statements already? 😱 I hope not. I'm pretty sure you can override whatever you are trying to override by adding a class or two extra. Ping me if you're in doubt.

Reply:
Unfortunately yes, since we have to override inline style.